### PR TITLE
added components Genres in common with styled

### PR DESCRIPTION
**Pull request by DorotaZielinska**
This is component used in TileList

### CODE REVIEW

File: src/common/Genres/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { GenresContainer } from "./styled";
```
Comment (MirekSobota) [inline src/common/Genres/index.js:1]: Shouldn't we import genres from './styled'? :) 

 And I'm not sure if it's correct to name the component and styled.li with the same name (Genres). Did you have any errors?

**Review by MirekSobota — APPROVED** [review]

File: src/common/Genres/index.js (hunk 1 above)
Comment (DorotaZielinska) [reply src/common/Genres/index.js:1]: Now I did,t any error, is in the same level Genres, and if is from another component then is other parts.

File: src/common/Genres/styled.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+
+export const GenresContainer = styled.div`
+    display: flex;
+    flex-wrap: wrap;
+    list-style: none;
+    padding-left: 0px;
+    margin: -8px;
```
Comment (PawelOwiesek) [inline src/common/Genres/styled.js:8]: Hey Dorota. Why do we need negative value of margin hier?

**Review by PawelOwiesek — COMMENTED** [review]
Please check line 8 in GenresContainer

**Review by PawelOwiesek — APPROVED** [review]
Looks very good. Great job.

**Review by PawelOwiesek — APPROVED** [review]
Yes , this is ok.

**Pull request merged**